packages/compliance-tests/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "libp2p-interfaces-compliance-tests",
+  "version": "0.0.0",
+  "description": "Compliance tests for JS Libp2p interfaces",
+  "files": [
+    "src",
+    "dist"
+  ],
+  "eslintConfig": {
+    "extends": "ipfs"
+  },
+  "scripts": {
+    "lint": "aegir lint",
+    "dep-check": "aegir dep-check",
+    "test": "aegir test",
+    "test:node": "aegir test --target node",
+    "test:browser": "aegir test --target browser",
+    "release": "aegir release --no-test",
+    "release-minor": "aegir release --type minor --no-test",
+    "release-major": "aegir release --type major --no-test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/libp2p/js-libp2p-interfaces.git"
+  },
+  "keywords": [
+    "libp2p",
+    "interface"
+  ],
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/libp2p/js-libp2p-interfaces/issues"
+  },
+  "homepage": "https://github.com/libp2p/js-libp2p-interfaces#readme",
+  "dependencies": {
+    "abort-controller": "^3.0.0",
+    "abortable-iterator": "^3.0.0",
+    "aegir": "^33.2.2",
+    "chai": "^4.3.4",
+    "chai-checkmark": "^1.0.1",
+    "delay": "^5.0.0",
+    "it-goodbye": "^3.0.0",
+    "it-pair": "^1.0.0",
+    "it-pipe": "^1.1.0",
+    "libp2p-interfaces": "^0.11.0",
```

Comment:
@achingbrain how do you handle dependency updates of adjacent modules?

Reply:
Do you mean when sibling versions change within a monorepo?  Lerna takes care of all that.